packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -1085,13 +1118,18 @@ abstract class RenderObjectWidget extends Widget {
   @protected
   void updateRenderObject(BuildContext context, RenderObject renderObject) { }
 
+  /// A render object previously associated with this widget has been removed
```

Comment:
The widget can have many render objects associated with it because it can be inserted into the tree in many locations.